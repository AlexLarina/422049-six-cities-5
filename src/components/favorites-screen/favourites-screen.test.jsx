import React from "react";
import renderer from "react-test-renderer";
import FavoritesScreen from "./favorites-screen.jsx";

it(`Render FavoritesScreen`, () => {
  const tree = renderer
    .create(
        <FavoritesScreen/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
