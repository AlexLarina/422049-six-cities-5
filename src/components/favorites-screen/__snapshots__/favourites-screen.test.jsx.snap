// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FavoritesScreen 1`] = `
<div
  className="page"
>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link"
            href="/"
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="../img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          className="header__nav"
        >
          <ul
            className="header__nav-list"
          >
            <li
              className="header__nav-item user"
            >
              <a
                className="header__nav-link header__nav-link--profile"
                href="#"
              >
                <div
                  className="header__avatar-wrapper user__avatar-wrapper"
                  style={
                    Object {
                      "backgroundImage": "url(undefined)",
                    }
                  }
                />
                <span
                  className="header__user-name user__name"
                  onClick={[Function]}
                />
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <main
    className="page__main page__main--favorites"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites favorites--empty"
      >
        <h1
          className="visually-hidden"
        >
          Favorites (empty)
        </h1>
        <div
          className="favorites__status-wrapper"
        >
          <b
            className="favorites__status"
          >
            Nothing yet saved.
          </b>
          <p
            className="favorites__status-description"
          >
            Save properties to narrow down search or plan yor future trips.
          </p>
        </div>
      </section>
    </div>
  </main>
  <footer
    className="footer container"
  >
    <a
      className="footer__logo-link"
      href="main.html"
    >
      <img
        alt="6 cities logo"
        className="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>
</div>
`;
